### DISCUSSION TITLE
Blue Yeti Nano experience?

### DISCUSSION THREAD

**Discussion by FrostKiwi** [jamulussoftware/jamulus#1392] [Technical Support & Help] [answered]
The [docs](https://jamulus.io/kb/2021/01/05/Jamulus-Sound-Devices.html) state, that the Blue Yeti is confirmed to work well.
Anyone happen to have a Blue Yeti Nano to confirm, whether the same is true for that cheaper variant?

**Comment by chrisrimple** [comment]
I use a Blue Yeti (and may have been the one to add it to the list).  No reason to believe the Nano will be any different - they're both just USB mics.

**Reply by FrostKiwi** [reply]
We are testing some stuff, before mass buying for our smallish orchestra.
At home, we have remote music teaching setups, which use the onboard soundcard as mic inputs. Using the ASIO4all driver, those onboard inputs provide 16-17ms latency.
Interestingly enough, the Blue Yeti nano always hovers around 30-33ms of delay, twice as much.
Before the Blue Yet Nano, we tried the Behringer Xenyx 302 USB, with the exact same driver. The numbers were exactly identical, 30-33ms as well.
Another person though reported 15ms of delay with the blue yeti.  All machines in question are Windows 10. There must be some secret magic electronic sauce, which determines what USB port is worthy of the divine low latency blessing.

The live monitoring of the blue yeti can be turned off, by pressing the cardiod/omni switch button on the back for 2 seconds, no need to install the manufacturer's "Sherpa" software, as the Jamulus doc suggests. For non-techies, the Blue Yeti is definitely nice with it's simpler setup, but the variance in delay is strange.

**Reply by chrisrimple** [reply]
Does the Yeti have it's own ASIO driver?  If so, try using that instead of ASIO4ALL.  I'm on a Mac, so no ASIO drivers required.

**Reply by FrostKiwi** [reply]
No, only ASIO4All. My take on Jamulus hardware would be: If you are on windows, do not buy the BlueYeti or the Behringer Xenyx 302 USB. The onboard soundcard was always lower latency, as tested on multiple PCs by different people. So get something like the Rode VideoMic Go, which is our setup here. That mic lives off of the Plug-In voltage from the soundcard and provides good sound after being gain boosted by the soundcard. Anything else seems too expensive...

**Comment by FrostKiwi** [comment, marked as answer]
So here is the experience:
On Windows, not good. We double checked and the 15ms latency was a miss-communication, the person confused server latency with the hardware latency.
On multiple computers, the latency never drops below 30ms, not matter (Buffer size, hardware buffer yes/no) what is done and usually hovers around 35-40ms. This was tested on multiple windows 10 revision, different motherboards, including one high-end workstation with windows's new 20H2 update.
The onboard soundcard with a mic plugged in always performs better (15-20ms) than any Devise using the ASIO4all driver. In that sense, the yeti mic is not at fault here, but probably the ASIO4all driver on windows.

**Reply by dcorson-ticino-com** [reply]
My advice is an audio box 24bit, 192kHz and a microphone.  
Those Yeti USB mics are more expensive than, for example an Behringer UMC202HD + a reasonable microphone.
I have a NI KompleteAudio 2, hardly more expensive, and am getting times of ping + 10 to 12ms, Win10 20H2.
(I have never gotten ASIO4ALL to work at all, it just gives me blue screens)

**Reply by FrostKiwi** [reply]
> 
> 
> My advice is an audio box 24bit, 192kHz and a microphone.
> Those Yeti USB mics are more expensive than, for example an Behringer UMC202HD + a reasonable microphone.
> I have a NI KompleteAudio 2, hardly more expensive, and am getting times of ping + 10 to 12ms, Win10 20H2.
> (I have never gotten ASIO4ALL to work at all, it just gives me blue screens)

That setup is surprisingly affordable. thanks for the reference.

**Reply by FrostKiwi** [reply]
@dcorson-ticino-com
Thx for the recommendation on the UMC202HD. With the manufacturer providing a ASIO driver, it turned out to be the least setup required in terms of fiddling with software and the least error prone. The latency as reported by jamulus on a server running on localhost (20-25ms), is still worse than using ASIO4ALL on the onboard soundcard (15-20ms), but with the driver's ease of use, it is the best solution yet. A German music equipment seller has a 125€ package for this USB-interface, a decent mic and headphones (which are tolerable for the duration of a music lesson)
In case of teachers playing with pupils: guiding pupils to port-forward the UDP port on the router once and shelling out for the 125€ package seems like the least time consuming solution so far. With pupils hosting on their laptop during the lesson and the teachers taking on the extra latency with their experience, this works out.
After trying a *lot* of equipment and a locally rented VPS for hosting, finally glad to have found something that works.
Unfortunately, with the way VDSL2 connections are in our state, pupils or small ensembles playing together is still a bit of a pipe dream.

**Reply by dcorson-ticino-com** [reply]
Thanks for the feedback Kiwi !
